src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_params.py

```diff
@@ -68,9 +68,15 @@
                       type=file_type, completer=FilesCompleter())
 
 with VersionConstraint(ResourceType.MGMT_RESOURCE_POLICY, min_api='2016-12-01') as c:
+    from azure.mgmt.resource.policy.models import PolicyMode
     c.register_cli_argument('policy definition', 'params',
                             help='JSON formatted string or a path to a file or uri with parameter definitions',
                             type=file_type, completer=FilesCompleter())
+    c.register_cli_argument('policy definition create', 'mode',
+                            options_list=('--mode', '-m'),
+                            help='mode of the new policy definition. One of Indexed or All',
```

Comment:
Now that you've added the enum_choice_list, you shouldn't need `One of Indexed or All`.

Reply:
done